test/loader_api.cpp

```diff
@@ -143,6 +143,19 @@ TEST(
   EXPECT_EQ(ZE_RESULT_SUCCESS, zeInit(ZE_INIT_FLAG_GPU_ONLY));
 }
 
+TEST(
+    LoaderInit,
+    GivenLevelZeroLoaderPresentWhenCallingZeInitDriversThenzeInitWithzeInitDriversUnsupportedOnTheDriverThenzeInitSucceeds) {
```

Comment:
I think a better name might be GivenZeInitDriversUnsupportedOnTheDriverWhenCallingZeInitDriversThenUninitializedReturned since that's the main point of the test.

Reply:
I will update the test name.